Makefile

```diff
@@ -0,0 +1,45 @@
+#  Copyright (c) University College London Hospitals NHS Foundation Trust
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+# limitations under the License.
+.PHONY: help
+
+SHELL:=/bin/bash
+MAKEFILE_FULLPATH := $(abspath $(lastword $(MAKEFILE_LIST)))
+MAKEFILE_DIR := $(dir $(MAKEFILE_FULLPATH))
+
+target_title = @echo -e "🌱$(1)..."
+
+all: deploy
+
+help: ## Show this help
+	@echo
+	@grep -E '^[a-zA-Z_-]+:.*?## .*$$' $(MAKEFILE_LIST) \
+		| awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%s\033[0m|%s\n", $$1, $$2}' \
+        | column -t -s '|'
+	@echo
+
+deploy: build ## Deploy this app
```

Comment:
I feel like perhaps this should be split into two commands - serve for building and running locally, and deploy for pushing to ACR? Feels a bit weird to use `make deploy` for local testing

Reply:
hmm, I did think about this but thought I'd rather just run `make` and let the right thing happen. Happy to alter if you think "deploying locally" is more annoying than having separate targets. (also then what is the default target?)

Reply:
Yeah good point - perhaps in relation to my other comment if we have a vs code debugging profile then we can just steer people to that (F5) and then who cares what the command is called :D

Reply:
true true. have thought about it more and you're right though – will separate. thanks